src/bitmessagekivy/baseclass/myaddress.py

```diff
@@ -0,0 +1,211 @@
+            else:
+                meny.add_widget(ToggleBtn(active=True if is_enable == 'true' else False))
+            self.ids.ml.add_widget(meny)
+
+    def check_scroll_y(self, instance, somethingelse):
+        """Load data on Scroll down"""
+        if self.ids.refresh_layout.scroll_y <= -0.0 and self.has_refreshed:
+            self.ids.refresh_layout.scroll_y = 0.06
+            my_addresses = len(self.ids.ml.children)
+            if my_addresses != len(self.addresses_list):
+                self.update_addressBook_on_scroll(my_addresses)
+            self.has_refreshed = (
+                True if my_addresses != len(self.addresses_list) else False
+            )
+
+    def update_addressBook_on_scroll(self, my_addresses):
+        """Loads more data on scroll down"""
+        self.set_mdList(my_addresses, my_addresses + 20)
+
+    def myadd_detail(self, fromaddress, label, *args):
+        """Load myaddresses details"""
+        if BMConfigParser().get(fromaddress, 'enabled') == 'true':
+            obj = MyaddDetailPopup()
+            self.address_label = obj.address_label = label
+            self.text_address = obj.address = fromaddress
+            width = .9 if platform == 'android' else .6
+            self.myadddetail_popup = self.myaddress_detail_popup(obj, width)
+            self.myadddetail_popup.auto_dismiss = False
+            self.myadddetail_popup.open()
+        else:
+            width = .8 if platform == 'android' else .55
+            self.dialog_box = self.inactive_address_popup(width, self.callback_for_menu_items)
+            self.dialog_box.open()
+
+    def callback_for_menu_items(self, text_item, *arg):
+        """Callback of inactive address alert box"""
+        self.dialog_box.dismiss()
+        toast(text_item)
+
+    def refresh_callback(self, *args):
+        """Method updates the state of application,
+        While the spinner remains on the screen"""
+        def refresh_callback(interval):
+            """Method used for loading the myaddress screen data"""
+            state.searcing_text = ''
+            self.ids.search_bar.ids.search_field.text = ''
+            self.has_refreshed = True
+            self.ids.ml.clear_widgets()
+            self.init_ui()
+            self.ids.refresh_layout.refresh_done()
+            Clock.schedule_once(self.address_permision_callback, 0)
+        Clock.schedule_once(refresh_callback, 1)
+
+    @staticmethod
+    def filter_address(address):
+        """Method will filter the my address list data"""
+        if [
+                x for x in [
+                    BMConfigParser().get(address, 'label').lower(),
+                    address.lower()
+                ]
+                if (state.searcing_text).lower() in x
+        ]:
+            return True
+        return False
+
+    def disableAddress(self, address, instance):
+        """This method is use for disabling address"""
+        BMConfigParser().set(str(address), 'enabled', 'false')
```

Comment:
split into frontend and backend. Same problem exists in `bitmessageqt/__init__.py`. In addition to that, hhere missing `reloadMyAddressHashes()`.